### PR TITLE
Google and SUS plugins

### PR DESCRIPTION
**Pull request by 0x454d505459**
Added two "useful" plugins:
- google 
- sus

### CODE REVIEW

**Comment by ActuallyTheSun** [comment]
the sus plugin is the most useful plugin added so far

(correction: 2nd most useful after 🗿)

**Comment by MaiKokain** [comment]
add your author info to [src/utils/constants.ts](https://github.com/Vendicated/Vencord/blob/39a7b2f5a95c7330730bc5c883e5eca7e39b377e/src/utils/constants.ts) instead, import it and use it. So you don't have to write your name and user id for every plugin you make/have been credited in.

**Comment by 0x454d505459** [comment]
> add your author info to [src/utils/constants.ts](https://github.com/Vendicated/Vencord/blob/39a7b2f5a95c7330730bc5c883e5eca7e39b377e/src/utils/constants.ts) instead, import it and use it. So you don't have to write your name and user id for every plugin you make/have been credited in.

Done

**Comment by Vendicated** [comment]
sus plugin is very useless so please remove it, the google one should url encode the input and please fix the typos

File: src/utils/constants.ts [hunk 1]
```diff
@@ -42,5 +42,9 @@ export const Devs = Object.freeze({
     Animal: {
         name: "Animal",
         id: 118437263754395652n
+    },
+    Empty: {
+        name: "EMPTY",
+        id: 382960284135849984
```
Comment (Cynosphere) [inline src/utils/constants.ts:48]: Missing BigInt notation (`n`) on ID (thus why linter is complaining)

**Review by Cynosphere — CHANGES_REQUESTED** [review]

**Comment by 0x454d505459** [comment]
Closing PR, both plugins are more than useless. Sorry to have wasted your time.

**Comment by emir4169** [comment]
> Closing PR, both plugins are more than useless. Sorry to have wasted your time.

WRONG.
OBJECTION! The Google Plugin is not useless! It clearly has a use.

**Pull request closed without merging**